src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -135,7 +141,9 @@ func runClientProxy(cfg *config.Config) {
 		ui.Show()
 	}
 
+	go logging.Configure(cfg)
```

Comment:
Using a goroutine because we'll be using waitforserver and at this time the proxy is not yet ready.